src/WebJobs.Extensions.DurableTask/EndToEndTraceHelper.cs

```diff
@@ -61,11 +63,11 @@ public static string LocalSlotName
                 instanceId, reason, functionType.ToString(), isReplay);
```

Comment:
Please add the `ExtensionVersion` data to the ETW trace so that folks on servicing can easily get this critical information.